thehive4py/api.py

```diff
@@ -329,6 +329,27 @@ def create_case_observable(self, case_id, case_observable):
             except requests.exceptions.RequestException as e:
                 raise CaseObservableException("Case observable create error: {}".format(e))
 
+
+    def delete_case_observable(self, observable_id):
+        """
+        Deletes a TheHive case observable.
+
+        Arguments:
+            observable_id (str): Id of the observable to delete
+
+        Returns:
+            response (requests.Response): Response object including true or false based on the action's success
+
+        Raises:
+            CaseException: An error occured during observable deletion
+        """
+        req = self.url + "/api/case/artifact/{}".format(observable_id)
+
+        try:
+            return requests.delete(req, proxies=self.proxies, auth=self.auth, verify=self.cert)
+        except requests.exceptions.RequestException as e:
+            raise CaseException("Observable deletion error: {}".format(e))
```

Comment:
Hello could you please raise a `CaseObservableException` instead?

Reply:
Hello, of course!